spec/unit/installer/xcode/pods_project_generator/pod_target_integrator_spec.rb

```diff
@@ -12,33 +12,32 @@ class PodTargetIntegrator
               @target_definition = fixture_target_definition
 
               @watermelon_spec = fixture_spec('watermelon-lib/WatermelonLib.podspec')
-              @watermelon_pod_target = fixture_pod_target_with_specs([@watermelon_spec, *@watermelon_spec.recursive_subspecs],
-                                                                     false, {}, [], Platform.ios, [@target_definition])
+              @watermelon_pod_target = fixture_pod_target_with_specs([@watermelon_spec,
+                                                                      *@watermelon_spec.recursive_subspecs], true, {},
+                                                                     [], Platform.ios, [@target_definition])
 
               @coconut_spec = fixture_spec('coconut-lib/CoconutLib.podspec')
               @coconut_pod_target = fixture_pod_target_with_specs([@coconut_spec, *@coconut_spec.recursive_subspecs],
-                                                                  false, {}, [], Platform.ios, [@target_definition])
+                                                                  true, {}, [], Platform.ios, [@target_definition])
 
               @native_target = stub('NativeTarget', :shell_script_build_phases => [], :build_phases => [],
                                                     :project => @project)
               @test_native_target = stub('TestNativeTarget', :symbol_type => :unit_test_bundle, :build_phases => [],
                                                              :shell_script_build_phases => [], :project => @project,
                                                              :name => 'CoconutLib-Unit-Tests')
 
-              @target_installation_result = TargetInstallationResult.new(@coconut_pod_target, @native_target, [],
-                                                                         [@test_native_target])
+              @coconut_target_installation_result = TargetInstallationResult.new(@coconut_pod_target, @native_target,
+                                                                                 [], [@test_native_target])
             end
 
             describe '#integrate!' do
-              it 'integrates test native targets with frameworks and resources script phases' do
-                PodTargetIntegrator.new(@target_installation_result).integrate!
-                @test_native_target.build_phases.count.should == 2
+              it 'integrates test native targets with framework script phase' do
+                PodTargetIntegrator.new(@coconut_target_installation_result).integrate!
+                @test_native_target.build_phases.count.should == 1
```

Comment:
this is correct. There are only frameworks to embed here but no resources since there are no resource bundles to copy.